.vsts-ci/sshremoting-tests.yml

```diff
@@ -0,0 +1,97 @@
+name: PR-$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:.rr)
+trigger:
+  # Batch merge builds together while a merge build is running
+  batch: true
+  branches:
+    include:
+    - master
+    - release*
+    - feature*
+  paths:
+    include:
+    - '/src/System.Management.Automation/engine/*'
+    - '/test/*'
+    exclude:
+    - /tools/releaseBuild/**/*
+    - /.vsts-ci/misc-analysis.yml
+    - /.github/ISSUE_TEMPLATE/*
+    - /.dependabot/config.yml
+pr:
+  branches:
+    include:
+    - master
+    - release*
+    - feature*
+  paths:
+    include:
+    - '/src/System.Management.Automation/engine/*'
+    - '/test/*'
+    exclude:
+    - /tools/releaseBuild/**/*
+    - /.vsts-ci/misc-analysis.yml
+    - /.github/ISSUE_TEMPLATE/*
+    - /.dependabot/config.yml
```

Comment:
you only need to exclude things that are under your include.  This can be removed.
```suggestion
```